src/main/java/org/selenium/pages/CartPage.java

```diff
@@ -0,0 +1,46 @@
+package org.selenium.pages;
+
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.selenium.ojb.Product;
+
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.openqa.selenium.support.ui.ExpectedConditions.visibilityOfElementLocated;
+
+public class CartPage extends BasePage {
+    private By cartButton = By.xpath("//a[@title='View my shopping cart']");
+    private By waitContainer = By.xpath("//div[@id='center_column']");
+    private By cartTotal = By.xpath("//span[@id='total_price']");
+    private By rows = By.xpath("//table/tbody/tr");
+    private List<Product> cartList = new ArrayList<>();
```

Comment:
I don't see the reason to have it on class level. Should be moved to the method level

Reply:
Fix it